src/features/calendar/components/CalendarMonthView/Day.tsx

```diff
@@ -23,6 +27,13 @@ const Day = ({
 }: DayProps) => {
   const isToday = dayjs(date).isSame(new Date(), 'day');
 
+  const dstOffset = useMemo(
+    () =>
+      getDSTOffset(dayjs(date).startOf('day').toDate()) -
+      getDSTOffset(dayjs(date).endOf('day').toDate()),
+    [date]
+  );
```

Comment:
This pattern returns three times. Maybe it could be a utility function that returns a boolean, which would also take a way the need of comparing `dstOffset !== 0` and have the added benefit of better describing what it is. The `dstOffset` variable name is correct in that it describes what the numeric value constitutes, but it doesn't really help me understand why that's relevant (or why `zero` is a special value on line 71). Something like this would maybe be cool?

```suggestion
  const dstChangesToday = useMemo(
    () => isDstChangeDay(date),
    [date]
  );
```